packages/odyssey-design-tokens/src/focus.json

```diff
@@ -1,9 +1,18 @@
 {
   "focus": {
     "outline": {
+      "color": {
+        "primary": { "value": "{color.primary.base.value}" },
+        "danger": { "value": "{color.danger.base.value}" }
+      },
+      "offset": {
+        "base": { "value": "2px" },
+        "tight": { "value": "0" }
```

Comment:
I am not at all married to this naming.

Reply:
I'd love to see everything unified. Perhaps that means using the numeric scale that seems to be in favor in your spacing updates PR. Starting from 0 or maybe 1? One benefit is we are freed from having to invent other adjectives to describe the delta between token values "base" vs "tight" etc.

Lets pick something and be consistent as much as we can!

Reply:
**Re: Naming**

For ordinal tokens like the color, space, and font-size scales, I think our 0-index list works well.

For these functional/specific tokens, it'd be useful to align on descriptive naming where we can. For ex, if we did build a FocusRing component, we would want a prop like `focusStyle="compact"` rather than `focusStyle="2"`.

Ideally, we'll use the same language with design as well. @josesolano-okta @gretchen-okta do you have any thoughts on naming?

**Re: Organization**

Maybe it would be better to reorganize these to represent the variant style they apply to? For ex: `focus.ring.offset` and `focus.compact.color` rather than `focus.offset.base` and `focus.width.tight` It's unlikely that someone will use these specific tokens outside of these contexts.

Reply:
the reason we use numbers is because there is a large list of tokens in a category. But for these there will never be more than the 2 options so I think it's fine to name them descriptively.